### ISSUE TITLE
DeepCluster on CIFAR10

### ISSUE THREAD

**Issue by hxixixh** [Raymond-sci/AND#5]
In the paper, it seems that DeepCluster method works well on CIFAR10, while in this issue [https://github.com/facebookresearch/deepcluster/issues/22](url) of DeepCluster, it works bad on a backbone of AlexNet. 

So I guess there are some tricks of training DeepCluster on CIFAR10 using AlexNet?

**Comment by Raymond-sci** [comment]
We used the original implementation from the mentioned repository with minor modifications for small-scale datasets but no extra trick was applied if I recall right. One thing is we set the number of clusters K to a very large value as how the DeepCluster did on ImageNet (way larger than the ground-truth one) but I failed to recall the exact number, should be like 100 for cifar10.

**Comment by hxixixh** [comment]
Could you provide the reproduce code? I have tried different learning rate and number of clusters, yet still got nmi lower than 0.1.

**Comment by Raymond-sci** [comment]
I would like to do so if possible but I afraid that the code is no longer there due to regular workspace cleanup. Feel free to reopen this issue if needed and I will see if I can manage time to reproduce it again.
